minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -724,6 +727,12 @@ private synchronized Worker createWorker(int i) throws IOException {
     conf.put(PropertyKey.LOGS_DIR, logsDir.getAbsolutePath());
     conf.put(PropertyKey.WORKER_RPC_PORT, rpcPort);
     conf.put(PropertyKey.WORKER_WEB_PORT, webPort);
+    conf.put(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_DIR, workerMetaStoreDir.getAbsolutePath());
+
+//    // Enable dora by adding the following config keys.
+//    conf.put(PropertyKey.DORA_CLIENT_UFS_ROOT, PathUtils.concatPath(mWorkDir, "underFSStorage"));
+//    conf.put(PropertyKey.DORA_CLIENT_READ_LOCATION_POLICY_ENABLED, true);
+//    conf.put(PropertyKey.MASTER_WORKER_REGISTER_LEASE_ENABLED, false);
```

Comment:
dora will be future default? so probably better to be default, but it's a cleanup task lol

Reply:
I tried to enable Dora in the UT. But there are several test failures. It needs some more work.
I will create a separate task for this.

Reply:
For example, the `alluxio.server.ft.ZookeeperFailureIntegrationTest.zkFailure` should be `Ingored` because Dora client does not rely on Master/Zookeeper.